docs/static/breaking-changes.asciidoc

```diff
@@ -34,6 +34,17 @@ These changes can impact any instance of Logstash and are plugin agnostic, but o
 * Configurations provided with `-f` or `config.path` will not be appended with `stdin` input and `stdout` output automatically.
 
 [float]
+
+==== Elasticsearch output changes
+
+* The default `document_type` used has changed from `logs` to `doc`. 
+  Furthermore, users are advised that Elasticsearch 6.0 deprecates doctypes, and 7.0 will remove them. 
+  See https://www.elastic.co/guide/en/elasticsearch/reference/master/removal-of-types.html[Removal of Mapping Types] for more info.
+* The options `:flush_size; `:idle_flush_time` are obsolete and may no longer be used
```

Comment:
Change to say: The options `flush_size` and `idle_flush_time` are now obsolete and have been removed. 

("may no longer be used" might be misinterpreted because people often confuse the meaning of may vs might)

Note the change to the formatting of the options, too.

Reply:
I think we can just say "are obsolete" -- https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/lib/logstash/outputs/elasticsearch/common_configs.rb#L95-L97

They're "present" in the plugin as defined, but marked as "obsolete" so a user trying to use them will crash Logstash on startup and Logstash will tell them "Hey this is obsolete and you cannot use it"